src/common/transformations/src/transformations/op_conversions/convert_gelu.cpp

```diff
@@ -4,34 +4,33 @@
 
 #include <memory>
 #include <ngraph/ngraph.hpp>
-#include <ngraph/opsets/opset2.hpp>
 #include <ngraph/pattern/op/wrap_type.hpp>
 #include <ngraph/rt_info.hpp>
+#include <openvino/opsets/opset1.hpp>
```

Comment:
Unnecessary

Reply:
I disagree, this `opset` is referred in there. Although it compiles without this direct inclusion, implicit `#include` might break.